### ISSUE TITLE
getArrows from Arrow's extension is not working

### ISSUE THREAD

**Issue by NishantDesai1306** [shaack/cm-chessboard#91]
The way _registerMethod_ is implemented any extension which "returns" a value will not be returns to the invocation call.
```
registerMethod(name, callback) {
        if (!this.chessboard[name]) {
            this.chessboard[name] = (...args) => {
                callback.apply(this, args) // missing return statement
            }
        } else {
            log.error("method", name, "already exists")
        }
    }
```

Right now after registering **Arrows** extension if implement something like this

```
const getAllArrows = (chessboard) => {
   const arrows = chessboard.getArrows();
   console.log(arrows) // this logs undefined right now
   return arrows;
}
```

Issue is reproducible in examples mentioned in the repository
![image](https://user-images.githubusercontent.com/18420157/199360548-c73a6f7d-df15-440a-905f-8b22f871dde8.png)
![image](https://user-images.githubusercontent.com/18420157/199360670-023508b3-c938-4b22-90aa-7bada6eca95a.png)

**Comment by shaack** [comment]
Fixed with v5.2.5. Thank you for the report.

**Comment by NishantDesai1306** [comment]
Thanks for quickly resolving it 👍
